sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -203,4 +204,9 @@ private ProxyHandler getProxyHandler(AuthorizationChallengeHandler challengeHand
                     String.format(INVALID_PROXY_MESSAGE, proxyOptions.getType())));
         }
     }
+
+    public NettyAsyncHttpClientBuilder disableBufferCopy() {
```

Comment:
Missing Javadoc

Reply:
will do all the javadoc, checkstyle, spotbugs stuff once we are happy with the shape of the code